### ISSUE TITLE
Custom collections do not work

### ISSUE THREAD

**Issue by adjabaev** [nuxt/icon#252]
Hi, hope you're all doing well!

I ran into an issue, I am using a custom iconify collection i built which goes by
"@iconify-json/as": "npm:@my-org-name/my-icon-collection@^0.0.2",
in my package.json which isn't referenced in
https://github.com/nuxt/icon/blob/main/src/collection-names.ts
which makes it uncompatible with @nuxt/icon.
I think the issue comes from there:
https://github.com/nuxt/icon/blob/cc14fe49b19fce023ba6f049984317492043c9ab/src/collections.ts#L72-L73
Would it be possible further expand the compatibility to include custom collections? (and eventually check if these are installed print an error on the logger if it isn't the case)

Thank you in advance! :)

**Comment by antfu** [comment]
Yeah I think that are the ways to use it. When `ssr: false` and `provider` is not set to `server`, the client will go to Iconify's API endpoint to ask for icons, which they don't have your custom iconset. The only way seems to serve if from your own server
